File: fixtures/elixir_output/post_form.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://domain.tld/post-to-me.php",
  options: [],
  headers: [],
  params: [],
  body: {:multipart, [
    {~s|username|, ~s|davidwalsh|},
    {~s|password|, ~s|something|}
]}
}

response = HTTPoison.request(request)

File: fixtures/php/post_form.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://domain.tld/post-to-me.php');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'POST');
curl_setopt($ch, CURLOPT_POSTFIELDS, [
	'username' => 'davidwalsh',
	'password' => 'something',
]);

$response = curl_exec($ch);

curl_close($ch);
